tests/suites/os/tests/healthcheck/index.js

```diff
@@ -30,18 +30,39 @@ module.exports = {
     },
   },
   run: async function (test) {
+    await delay(1000 * 60);
```

Comment:
I think we can remove this now that https://github.com/balena-os/meta-balena/pull/2245 has been merged